bluepyopt/ephys/locations.py

```diff
@@ -259,6 +259,7 @@ class NrnSomaDistanceCompLocation(Location, DictMixin):
     """Compartment at distance from soma"""
 
     SERIALIZED_FIELDS = ('name', 'comment', 'soma_distance', 'seclist_name', )
+    max_diam = 0.0
```

Comment:
This is a class variable. Are you sure you want to share this between all instances of that class?
What's the function of changing this?

Reply:
I do not know why we need the check 
`if comp_diam > max_diam:`

Is there was a reason why it is:
```
max_diam = 0
if comp_diam > max_diam:
```
and not ?
`if comp_diam > 0:`


But indeed class variable might not be the proper way.

Reply:
Yes, that's true. Not sure where that logic came from originally, it was probably used to avoid zero-diameter sections. We can use if comp_diam > 0

Reply:
Changed !